accepted/PSR-16-simple-cache-meta.md

```diff
@@ -68,3 +68,26 @@ For being an early reviewer
 * [Survey of existing cache implementations][1], by @dragoonis
 
 [1]: https://docs.google.com/spreadsheet/ccc?key=0Ak2JdGialLildEM2UjlOdnA4ekg3R1Bfeng5eGlZc1E#gid=0
+
+## 8. Errata
+### 8.1 Type additions
+
+The 2.0 release of the `psr/simple-cache` package includs scalar parameter types and increases the minimum PHP version to 8.0 respectively.  This is considered a backwards compatible change for implementing libraries as PHP 7.2 introduces covariance for parameters.  Any implementation of 1.0 is compatible with 2.0. For calling libraries, however, this reduces the types that they may pass (as previously any parameter that could be cast to string could be accepted) and as such requires incrementing the major version.
+
+The 3.0 release includes return types.  Return types break backwards compatibility for implementing libraries as PHP does not support return type covariance.
```

Comment:
This is not true.  PHP supports return type contravariance (or whatever the opposite of what parameters do is. I can never remember which is which.)  A child can add a more restrictive return type than its parent, but cannot widen it.

Reply:
```suggestion
The 3.0 release includes return types.  Return types break backwards compatibility for implementing libraries as PHP does not support return type widening.
```